core/types/proto_block.proto

```diff
@@ -104,6 +104,21 @@ message ProtoWorkObject {
 
 message ProtoWorkObjects { repeated ProtoWorkObject work_objects = 1; }
 
+message ProtoWorkObjectBlockView {
+    optional ProtoWorkObjectHeader wo_header = 1;
+    optional ProtoWorkObjectBody wo_body = 2;
+}
+
+message ProtoWorkObjectHeaderView {
+    optional ProtoWorkObjectHeader wo_header = 1;
+    optional ProtoWorkObjectBody wo_body = 2;
+}
+
+message ProtoWorkObjectShareView {
```

Comment:
why is this needed?

Reply:
This is consumed by WorkObjectHeader so no need, I will remove it.